ONSdigital/dp-search-api
# Feature/upgrade search client

### What

Replaced references to rchttp with net, added environment variable for signing requests. 

### How to review

- see all mention of dp-rchttp removed from application and replaced with dp-net and dp-elasticsearch
- see that signing of elasticsearch requests is being implemented
- [(https://trello.com/c/Uft0380g/752-search-api-upgrade-search-client)]

### Who can review

Anyone but me!
